src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/ApiTests/InstanceApiPlatformFailsTest.cs

```diff
@@ -31,7 +31,7 @@ public async Task Instance_Get_FailsOK()
 
             HttpResponseMessage response = await client.SendAsync(httpRequestMessage);
             
-            Assert.Equal(HttpStatusCode.BadRequest, response.StatusCode);
+            Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
```

Comment:
Hmm, does this seem reasonable? That we go from 400 to 500 by simply changing the authorization ?

Reply:
Removed test